products/bridge/smart-contracts/contracts/periphery/BridgedToken.sol

```diff
@@ -16,6 +16,26 @@ interface IBridgedToken is IERC20 {
 
 contract BridgedToken is IERC20, ERC20, ERC20Burnable, Ownable {
```

Comment:
:+1: - or remove IBridgedToken

Reply:
Also, traditional to use Ownable2Step here, rather than Ownable - Ownable leaves you in the state where it is possble to accidentally renounce ownership (or transfer it to someone who doesn't exist) - this may be intentional, of course, in which case you should say so in comments.

Reply:
@rrw-zilliqa this is simply the standard way Ownership works in solidity. This just using OpenZeppelin's implementation for access control which is industry standard now. I see the improvements in ergonomics tho. I don't think we will have time to implement Ownable2Step here tho.